src/main/java/org/folio/linked/data/service/impl/ResourceServiceImpl.java

```diff
@@ -288,4 +280,17 @@ private Resource getResource(Long id) {
     return resourceRepo.findById(id)
       .orElseThrow(() -> new NotFoundException(RESOURCE_WITH_GIVEN_ID + id + IS_NOT_FOUND));
   }
+
+  private Resource toNewResource(ResourceRequestDto resourceDto, Resource old) {
+    var mapped = resourceDtoMapper.toEntity(resourceDto);
+    return addInternalFields(old, mapped);
+  }
+
+  private Resource addInternalFields(Resource oldResource, Resource newResource) {
```

Comment:
suggest to make it `void`
it always returns the same object, but it is confusing returned value